app/_src/gateway/plugin-development/get-started/testing.md

```diff
@@ -0,0 +1,364 @@
+custom plugins. 
+
+## Prerequisites
+
+This page is the second chapter in the [Getting Started](/gateway/{{page.gateway_release}}/plugin-development/get-started/index) 
+guide for developing custom plugins. These instructions refer to the previous chapter in the guide and require the same
+developer tool prerequisites.
+
+## Step by step
+
+Now that you have a basic plugin project, you can build testing automations for it.
+
+### Install Pongo
+
+[Pongo](https://github.com/Kong/kong-pongo) is a tool that helps you validate and 
+distribute custom plugins for {{site.base_gateway}}. Pongo uses Docker to
+bootstrap a {{site.base_gateway}} environment allowing you to quickly load your plugin, 
+run automated testing, and manually validate the plugin's behavior against
+various {{site.base_gateway}} versions.
+
+The following script can automate the installation of Pongo for you. 
+If you prefer, you can follow the [manual installation instructions](https://github.com/Kong/kong-pongo?tab=readme-ov-file#installation)
+instead.
+
+If you already have Pongo installed, you can skip to the [next step](#initialize-the-plugin-environment) or run the install script
+to update Pongo to the latest version.
+
+Run the following to install or update Pongo:
+
+```sh
+curl -Ls https://get.konghq.com/pongo | bash
+```
+
+For the remainder of this guide to work properly, the `pongo` command must be present in your system
+path. The script and manual installation instructions above both include hints for
+putting `pongo` on your path. 
+
+Ensure that the `pongo` command is 
+available in your `PATH` by running the command within your project directory:
+
+```sh
+pongo help
+```
+
+With Pongo installed, you can now set up a test environment for your new plugin.
+
+### Initialize the test environment
+
+Pongo lets you validate a plugin's behavior by giving you tools to quickly run a 
+{{site.base_gateway}} with the plugin installed and available. 
+
+Let's validate the plugin 
+manually first, and then you will add automated tests in subsequent steps of this guide. 
+
+{:.note}
+> **Note**: {{site.base_gateway}} runs in a variety of
+> [deployment topologies](/gateway{{page.release}}/production/deployment-topologies). 
+> By default, Pongo runs {{site.base_gateway}} in _traditional mode_, which uses a database 
+> to store configured entities such as routes, services, and plugins. 
+> {{site.base_gateway}} and the database are run in separate containers,
+> letting you cycle the gateway independently of the database. This enables a quick and 
+> iterative approach to validating the plugin's logical behavior while keeping the gateway
+> state independent in the database.
+
+Pongo provides an optional command that initializes the project directory with some
+default configuration files. You can run it to start a new project.
+
+{:.important}
+> **Important:** These commands must be run from the base `my-plugin` directory so that Pongo properly 
```

Comment:
I'm fine with `inside` but there are two `my-plugin` directories. @lena-larionova pointed out it may be confusing, I was hoping the use of the term `base` conveyed the project root.  So maybe I should use "project root folder (my-plugin)" instead.

Reply:
please see revised version of this.  Thank you for the reviews @mheap and @lena-larionova